scripts/runalltests.sh

```diff
@@ -0,0 +1,37 @@
+workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/testslog
+testdir=$workdir/../../../tests
+
+if [ -d $workdir ]; then
```

Comment:
Ah, now I see. If you only use workdir to store a single temporary file, I think using something like
`logfile="$(mktemp)"` is much easier, better portable and does not break if you run it multiple times.